data/tsv/bel_cyrl_phonetic.tsv

```diff
@@ -1,5 +1,3 @@
-аб'явіць	a b j a vʲ i t͡sʲ
```

Comment:
The language selector should not be throwing these out. Apostrophe is part of the Belarusian orthography. It's used to separate a hard (non-palatalized) consonant from the following iotated vowel (otherwise it'd be soft), thus в'етнам is [vjetnam] 'Vietnam and not [vʲjetnam]. Is there some way to modify it to not exclude such words?